### PR TITLE
Some concepts and latest node versions (travis) added

### PR DESCRIPTION
**Pull request by joseluisq**
PR details:
- f5b67a2 : Some improvements like concepts and resources added to `README.md`.
- e567f02 : Added latest node versions (v5 and v6) to `.travis.yml`.
- 8e05b81 : Minor `README.md` fixes.
- e2b7287 : `package.json` updated to `v1.0.7`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Learn **T**est **D**riven **D**evelopment (**TDD**)
+[# Learn **T**est **D**riven **D**evelopment (**TDD**)
```
Comment (nelsonic) [inline README.md]: the square bracket at the start of the line looks strange...
https://github.com/joseluisq/learn-tdd/blob/e567f020410d512fd0d71bf1ac8f184131eba50b/README.md
were you going to add a link?

**Comment by nelsonic** [comment]
@joseluisq thanks *very* much for this *great* PR! I left you a comment regarding the square bracket on the first line... also, if you update the version in `package.json` to: `1.0.7` we can *publish* a new version to NPM. 👍

File: README.md (hunk 1 above)
Comment (joseluisq) [inline README.md]: Fixed 8e05b8113180c08e5581859edf21b86f44279b77

**Comment by joseluisq** [comment]
> @joseluisq thanks very much for this great PR! I left you a comment regarding the square bracket on the first line... also, if you update the version in package.json to: 1.0.7 we can publish a new version to NPM.

Done e2b7287 :+1:

**Comment by nelsonic** [comment]
Sweet. merging. 🚀

**Comment by joseluisq** [comment]
Great 👍

**Pull request merged**